website/docs/r/cse_insights_configuration.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_insights_configuration"
+description: |-
+    Provides the Sumologic CSE Insights Configuration for the whole organization. There can be only one configuration per organization.
+---
+
+# sumologic_cse_insights_configuration
+Provides the Sumologic CSE Insights Configuration for the whole organization. There can be only one configuration per organization.
+
+## Example Usage
+```hcl
+resource "sumologic_cse_insights_configuration" "insights_configuration" {
+  lookback_days = 13.0
+  threshold = 12.0
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `lookback_days` - (Optional) Detection window expressed in days.
+- `threshold` - (Optional) Detection threshold activity score.
+
+The following attributes are exported:
+
+- `ID` - The internal ID of the insights configuration.
+
+## Import
+
+Insights Configuration can be imported using the field id, e.g.:
+```hcl
+terraform import sumologic_cse_insights_configuration.insights_configuration ID
```

Comment:
Wouldn't ID always be "cse-insights-configuration"? The import section should be similar to Policies resource https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/policies#import